src/manifest.py

```diff
@@ -25,7 +25,7 @@
 import asyncio
 from enum import IntEnum
 import logging
-import os
+import errno, os, sys
```

Comment:
Not sure if this repo has any style preference but I usually use one import per line.